### PR TITLE
Add set network command and remove testnet options - Closes #477

### LINKED ISSUE #477: Add `set network` command
**Expected behavior:** `set network` should switch between:
- mainnet
- testnet
- betanet
- custom network specified by valid nethash

This should also change the shape of `default_config.json`.

**Actual behavior:** We only have a boolean for testnet/mainnet

### Versions

1.0.0

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
In order to support beta net option, we needed to add an option to set the network, instead of boolean testnet option.

### How did I fix it?
- Add `api.network` option to config
- Remove `testnet` options

### How to test it?
```
set api.network beta
get delegate genesis_1
```

### Review checklist

* The PR solves #477 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: src/commands/set.js [hunk 1]
```diff
@@ -98,9 +99,30 @@ const setString = (dotNotation, value) => {
...
 };
 
+const setNethash = (dotNotation, value) => {
+	if (
+		dotNotation === 'api.network' &&
+		!Object.keys(NETHASHES).includes(value)
+	) {
+		if (value.length !== 64) {
+			throw new ValidationError(
+				'Value must be a hex string with 64 characters.',
```
Comment (willclarktech) [inline src/commands/set.js]: Actually we should inform users they can use `main`, `test` and `beta` here.

File: src/commands/set.js [hunk 2]
```diff
@@ -98,9 +99,30 @@ const setString = (dotNotation, value) => {
...
+			);
+		}
+		try {
+			lisk.cryptography.hexToBuffer(value, 'utf8');
+		} catch (error) {
+			throw new ValidationError(
+				'Value must be a hex string with 64 characters.',
```
Comment (willclarktech) [inline src/commands/set.js]: Prefer storing this in a variable for reuse.

File: src/utils/api.js [hunk 3]
```diff
@@ -15,15 +15,21 @@
...
-const { APIClient } = lisk;
+const { APIClient, constants } = lisk;
 
-const getAPIClient = testnet => {
-	const testnetOverrideValue =
-		typeof testnet === 'boolean' ? testnet : config.api.testnet;
-	return testnetOverrideValue === true
-		? APIClient.createTestnetAPIClient(config.api)
-		: APIClient.createMainnetAPIClient(config.api);
+const addresses = {
```
Comment (willclarktech) [inline src/utils/api.js]: This is liable to cause confusion. How about `bootstrap_nodes`?

File: default_config.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
 	"name": "lisky",
 	"json": false,
 	"api": {
-		"testnet": false,
-		"node": ""
+		"node": "",
```
Comment (willclarktech) [inline default_config.json]: Should we actually be making this an array of strings?

File: test/specs/commands/set.js [hunk 5]
```diff
@@ -819,6 +815,112 @@ describe('set command', () => {
 									},
 								);
 							});
+							Given(
+								'a value "ed14889723f24ecc54871d058d98ce91ff2f973192075c0155ba2b7b70ad2500"',
```
Comment (willclarktech) [inline test/specs/commands/set.js]: It's a little confusing having a nethash that is so similar to mainnet. Can we have a totally different one?

File: default_config.json (hunk 4 above)
Comment (shuse2) [reply default_config.json]: nodes will be the best i think, but should we do it with this PR?

File: test/specs/commands/set.js (hunk 5 above)
Comment (shuse2) [reply test/specs/commands/set.js]: let's change it to unlikely happening value like `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`?

File: src/commands/set.js (hunk 1 above)
Comment (shuse2) [reply src/commands/set.js]: how about changing error to
`Value must be a hex string with 64 characters. Alternatively, "main", "test" and "beta" can be used`

File: src/commands/set.js [hunk 6]
```diff
@@ -25,12 +26,15 @@ const description = `Sets configuration <variable> to <value>. Variables availab
...
 	- set name my_custom_lisky
-	- set api.testnet true
+	- set api.network main
 `;
 
 const WRITE_FAIL_WARNING =
 	'Config file could not be written: your changes will not be persisted.';
 
+const NETHASH_ERROR_MESSAGE =
+	'Value must be a hex string with 64 characters. Alternatively, main, test and beta can be used.';
```
Comment (willclarktech) [inline src/commands/set.js]: Value must be a hex string with 64 characters, or one of `main`, `test` or `beta`.

File: src/utils/api.js (hunk 3 above)
Comment (willclarktech) [reply src/utils/api.js]: This still applies.

File: default_config.json (hunk 4 above)
Comment (willclarktech) [reply default_config.json]: https://github.com/LiskHQ/lisky/issues/481

**Pull request merged**